src/MapViewController.swift

```diff
@@ -74,6 +76,35 @@ public class MapViewController: TGMapViewController, LocationManagerDelegate, TG
     shouldFollowCurrentLocation = enabled
   }
 
+  public func loadScene(named: String, apiKey: String) {
+    self.loadSceneFile(named)
+    self.queueSceneUpdate("sources.mapzen.url_params", withValue: "{ api_key: \(apiKey)}")
+    self.applySceneUpdates()
+  }
+
+  public func add(annotations: [PeliasMapkitAnnotation]){
+    for annotation in annotations {
+      let newMarker = self.markerAdd()
+      markerSetPoint(newMarker, coordinates: TGGeoPoint(coordinate: annotation.coordinate))
+      markerSetStyling(newMarker, styling: "{ style: sdk-point-overlay, sprite: ux-search-active, size: [24, 36px], collide: false }")
+      currentAnnotations[annotation] = newMarker
+
+    }
+  }
+
+  public func remove(annotation: PeliasMapkitAnnotation) {
+    guard let markerId = currentAnnotations[annotation] else { return }
+    markerRemove(markerId)
```

Comment:
Just in case Tangram fails it would be good to check the return status of `markerRemove` before, I also suggests to return a boolean that gets this status from this function.